src/containers/Movie/Movie.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 
-const Movie = ({ poster_path }) => {
-  return (
-    <div className='movie-card'>
-      <img className='movie-image' src={`http://image.tmdb.org/t/p/w342/${poster_path}`} alt="a" />
-    </div>
-  )
+const Movie = (props) => {
+  const { id, location, poster_path } = props;
+  const singleMovie = location.pathname.includes(id)
+
+  if (singleMovie) {
+    return (
+      <div>
+        <h4>{props.id}</h4>
+        <h4>{props.title}</h4>
+        <h4>{props.overview}</h4>
+      </div>
+    )
+  } else {
+    return (
+      <div className='movie-card'>
+        <Link to={`/movies/${id}`}>
+          <img
+            className='movie-image'
+            src={`http://image.tmdb.org/t/p/w342/${poster_path}`}
+            alt="movie" />
+        </Link>
+      </div>
+    )
+  }
```

Comment:
This looks good. I did a fair amount of updating to the Movie container for the favorites functionality. Maybe we can move this to the MoveDetails component and have a Link in Movie that routes the user to the movie specific URL and displays the MovieDetails component onClick. I think MovieDetails could be a pop-up or remove all the other cards entirely.